dev/scripts/main.js

```diff
@@ -206,6 +210,25 @@ bikeApp.getDistanceDuration = function(stationOrigin, stationDestination) {
 	});
 }
 
+bikeApp.getDistanceDurationRoundTrip = function(stationOrigin, userDestination) {
+	var distanceService = new google.maps.DistanceMatrixService();
+	distanceService.getDistanceMatrix({
+	    origins: [stationOrigin],
+	    destinations: [userDestination],
+	    travelMode: google.maps.TravelMode.BICYCLING,
+	    unitSystem: google.maps.UnitSystem.METRIC,
+	    durationInTraffic: true
+
+	},
+	function (response, status) {
+	    if (status !== google.maps.DistanceMatrixStatus.OK) {
+	        console.log('Error:', status);
+	    } else {
+	    	bikeApp.roundTripTravelTime = Math.floor(((response.rows[0].elements[0].duration.value) / 60) * 2); //converting roundtrip travel time to minutes ( * 2 as it is a round trip)
```

Comment:
do not store 'roundTripTravelTime' in global variable should call the next function after this ajax call